src/spring-cloud/azext_spring_cloud/vendored_sdks/appplatform/v2022_01_01_preview/models/_models_py3.py

```diff
@@ -4060,6 +4062,7 @@ class JarUploadedUserSourceInfo(UploadedUserSourceInfo):
         'version': {'key': 'version', 'type': 'str'},
         'relative_path': {'key': 'relativePath', 'type': 'str'},
         'runtime_version': {'key': 'runtimeVersion', 'type': 'str'},
+        'jvm_options': {'key': 'jvmOptions', 'type': 'str'},
```

Comment:
Is this change missing in last PR for vendored SDK update?

Reply:
Yes, sad story, the missing is from swagger side. We have added it back